Sources/MapboxMaps/Viewport/ViewportImpl.swift

```diff
@@ -37,6 +37,12 @@ internal final class ViewportImpl: ViewportImplProtocol {
 
     private let anyTouchGestureRecognizer: UIGestureRecognizer
 
+    deinit {
+        currentCancelable?.cancel()
+        currentCancelable = nil
```

Comment:
We don't need this. 
I also saw Xcode 14 to warn about that line as something that would be an error in future versions